tests/blake2b.cc

```diff
@@ -38,21 +39,9 @@ using namespace std;
 
 // -----------------------------------------------------------------------------
 
-// API EXAMPLE HEX START
-void
-print_hex_ln(size_t bytes_len, uint8_t* bytes)
-{
-  for (int i = 0; i < bytes_len; ++i) {
-    printf("%02x", bytes[i]);
-  }
-
-  printf("\n");
-}
-// API EXAMPLE HEX END
-
 TEST(ApiTestSuite, ApiTest)
 {
-  // API EXAMPLE CODE START
+  // ANCHOR(example)
   // Reserve memory for a 64 byte digest, i.e.,
   // for a BLAKE2B run with full 512-bit output.
```

Comment:
The name is "BLAKE2b". This would also be consistent with the "BLAKE2s" used elsewhere.